<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1052</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>网格划分软件</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../ico/mainwindow_ico.ico</normaloff>../ico/mainwindow_ico.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1051</width>
      <height>521</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
    <property name="tabsClosable">
     <bool>true</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1052</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>　　文件　　</string>
    </property>
    <widget class="QMenu" name="menu_openfile">
     <property name="title">
      <string>打开</string>
     </property>
     <addaction name="action_open_dxf"/>
     <addaction name="action_open_img"/>
     <addaction name="separator"/>
     <addaction name="action_open_gridsta"/>
    </widget>
    <widget class="QMenu" name="menu_savefile">
     <property name="title">
      <string>保存</string>
     </property>
     <addaction name="action_save_img"/>
     <addaction name="separator"/>
     <addaction name="action_save_gridsta"/>
    </widget>
    <addaction name="menu_openfile"/>
    <addaction name="menu_savefile"/>
   </widget>
   <widget class="QMenu" name="menu_tools">
    <property name="title">
     <string>网格生成工具</string>
    </property>
    <addaction name="action_generateGrid_paint"/>
    <addaction name="action_generateGrid_dxf"/>
   </widget>
   <widget class="QMenu" name="menu_paint_tools">
    <property name="title">
     <string>　绘图工具　</string>
    </property>
    <widget class="QMenu" name="menu_paint">
     <property name="title">
      <string>绘制</string>
     </property>
     <addaction name="action_paint_line"/>
     <addaction name="action_paint_rect"/>
     <addaction name="action_paint_ellipse"/>
    </widget>
    <widget class="QMenu" name="menu_change">
     <property name="title">
      <string>调整</string>
     </property>
     <addaction name="action_change_width"/>
     <addaction name="action_change_color"/>
    </widget>
    <addaction name="menu_change"/>
    <addaction name="separator"/>
    <addaction name="menu_paint"/>
    <addaction name="separator"/>
    <addaction name="action_clearboard"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_paint_tools"/>
   <addaction name="menu_tools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar_showCC"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_open_dxf"/>
   <addaction name="action_open_img"/>
   <addaction name="separator"/>
   <addaction name="action_save_img"/>
   <addaction name="action_save_gridsta"/>
   <addaction name="separator"/>
   <addaction name="action_clearboard"/>
   <addaction name="separator"/>
   <addaction name="action_paint_line"/>
   <addaction name="action_paint_rect"/>
   <addaction name="action_paint_ellipse"/>
   <addaction name="separator"/>
   <addaction name="action_generateGrid_dxf"/>
   <addaction name="action_generateGrid_paint"/>
   <addaction name="action_generateGrid_img"/>
  </widget>
  <action name="action_open_dxf">
   <property name="text">
    <string>打开dxf文件</string>
   </property>
   <property name="toolTip">
    <string>打开dxf文件</string>
   </property>
  </action>
  <action name="action_open_gridsta">
   <property name="text">
    <string>打开网格数据</string>
   </property>
  </action>
  <action name="action_save_img">
   <property name="text">
    <string>保存图片</string>
   </property>
  </action>
  <action name="action_save_gridsta">
   <property name="text">
    <string>保存网格数据</string>
   </property>
  </action>
  <action name="action_generateGrid_paint">
   <property name="text">
    <string>生成绘图网格</string>
   </property>
  </action>
  <action name="action_open_img">
   <property name="text">
    <string>打开图片</string>
   </property>
  </action>
  <action name="action_generateGrid_dxf">
   <property name="text">
    <string>生成dxf文件网格</string>
   </property>
  </action>
  <action name="action_generateGrid_img">
   <property name="text">
    <string>生成输入图片网格</string>
   </property>
  </action>
  <action name="action_clearboard">
   <property name="text">
    <string>清空画板</string>
   </property>
  </action>
  <action name="action_paint_line">
   <property name="text">
    <string>线</string>
   </property>
  </action>
  <action name="action_paint_rect">
   <property name="text">
    <string>矩形</string>
   </property>
  </action>
  <action name="action_paint_ellipse">
   <property name="text">
    <string>椭圆</string>
   </property>
  </action>
  <action name="action_change_width">
   <property name="text">
    <string>画笔线宽</string>
   </property>
  </action>
  <action name="action_change_color">
   <property name="text">
    <string>画笔颜色</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
